### ISSUE TITLE
Video does not have a caption track.

### ISSUE THREAD

**Issue by HerinHentry** [mr-bobz/talarts#74]
Video does not have a caption track.
Issue tested and identified using aXe for firefox. This is reported as a Critical WCAG AA failure.
Our code:
...video class="video-stream html5-main-video" style="width: 931px; height: 570px; left: 0px; top: -570px; transform: none;"....

Sample track element:
...video width="320" height="240" controls>
...track src="subtitles_en.vtt" kind="subtitles" srclang="en" label="English">
.../video>

Note : Replacing tags with ... as it's a code snippet.

**Comment by christosptr** [comment]
@mr-bobz Hi Bobby is this possible to implemented with the YouTube player?

**Comment by mr-bobz** [comment]
We cannot change youtube code. Perhaps will have to look for alternative player then.

**Comment by mr-bobz** [comment]
@christosptr you can now add caption via the new Accessible Video Library plugin.

Look for Videos section similar to Posts and Pages in wp admin.

**Comment by christosptr** [comment]
Done

**Comment by HerinHentry** [comment]
Issue fixed. Excellent Effort Chris.
